### ISSUE TITLE
Missing structural elements (WCAG 1.3.1d)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1222]
**Request:** Paragraphs are marked with suitable structural elements. The division into smaller units facilitates handling and understanding. 

**Issues:**
Some elements of different pages are not marked with the necessary p-element:

1. https://slidewiki.org/

- "Fangen Sie sofort an. Meine Präsentationen." (missing p)

2. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/437649-3/?language=en-/slide-719796-3#/slide-437649-3

- Empty p-elements should be removed.
- List elements should not be additionally marked with the p element.

3. https://slidewiki.org/search?keywords=accessibility&sort=score

- text area below "Results" (missing p)

4. https://slidewiki.org/accessibility

- paragraphs within the boxes (missing p)

**Comment by VivienDZB** [comment]
I just noticed, that there is still a problem with missing p elements when creating a presentation: As soon as the creator doesn't set a style template in the editor but keeps the preset option "Format", no p element is created within the presentation. Is it somehow possible to still create the p-Element or to select the "Normal" style as default?

**Comment by aoelen** [comment]
I changed the default behaviour, on enter now a new paragraph is added instead of a div.
